test/package/test_package_fx.py

```diff
@@ -17,19 +17,31 @@
     from common import PackageTestCase
 
 
+class CustomTracer(Tracer):
+    def is_leaf_module(self, m, module_qualified_name,) -> bool:
+        return super().is_leaf_module(m, module_qualified_name) or module_qualified_name == "MyModule"
+
+
+class MyModule(torch.nn.Module):
+    def forward(self, x):
+        return torch.relu(x)
+
+
+class SimpleTest(torch.nn.Module):
+    def forward(self, x):
+        return torch.relu(x + 3.0)
+
+
 class TestPackageFX(PackageTestCase):
     """Tests for compatibility with FX."""
 
     def test_package_fx_simple(self):
-        class SimpleTest(torch.nn.Module):
-            def forward(self, x):
-                return torch.relu(x + 3.0)
-
         st = SimpleTest()
         traced = symbolic_trace(st)
 
         f = BytesIO()
         with PackageExporter(f, verbose=False) as pe:
+            pe.extern(["io", "sys"])
```

Comment:
I think these are needed in order to pickle the tracer?

Reply:
This comment (and the one about MyModule and SimpleTest being pickled) are vaguely concerning. Can you look at the package to see what's happening? I suspect we are packaging this test file which is not what we want; we should move the custom tracer and other test modules into their own modules if that's the case